src/core/time.d

```diff
@@ -611,7 +622,7 @@ public:
                   subtract this $(D Duration) from.
       +/
     Duration opBinaryRight(string op, D)(D lhs) const nothrow @nogc
-        if((op == "+" || op == "-") &&
+        if((op == "+" || op == "-" || op == "%") &&
             is(_Unqual!D == TickDuration))
     {
         return Duration(mixin("lhs.hnsecs " ~ op ~ " _hnsecs"));
```

Comment:
As above, plus I think the test coverage in core.time and std.datetime is quite over the top as it is. Adding tests here will not improve test coverage in any way.